admin/init.sls

```diff
@@ -0,0 +1,5 @@
+admin-packages:
+  pkg.installed:
+    - pkgs:
+      - mosh
```

Comment:
FYI, for now I'm just using the screen version that's pre-installed on OS X; it looks like there's a newer version available in Homebrew, but it's a little annoying to use from Salt. LMK if you need the newer version.